runtime/exports.t4

```diff
@@ -177,23 +177,23 @@
 
 		#region metadata/profiler.h
 
-		new Export ("void", "mono_profiler_set_events",
-			"MonoProfileFlags", "events"
+		new Export ("MonoProfilerHandle", "mono_profiler_install",
```

Comment:
This will cause Xamarin.Mac apps referencing the system mono to fail to load if using an older system mono, because the app won't find these symbols.

Please make these symbols optional by adding `true` as the first argument:

```csharp
    new Export (true, "MonoProfilerHandle", ...)
```

that way the app will only break when someone actually tries to profile as opposed to on startup.